install/Makefile.core.mk

```diff
@@ -20,7 +20,7 @@ GIT_COMMITSHA = $(shell git rev-list -1 HEAD)
 GIT_STRIPPED_VERSION=$(shell git describe --tags `git rev-list --tags --max-count=1` | cut -c 2-)
 REMOTE_PROVIDER="remote"
 LOCAL_PROVIDER="local"
-GOVERSION = 1.19.1
+# GOVERSION = 1.19.1
```

Comment:
Undo this change, please. Are you not using golang 1.19 on your dev system?

Reply:
Yes/no?

Reply:
Made this change. This was mistake